pkg/pool-stable-phantom/contracts/StablePoolProtocolFees.sol

```diff
@@ -96,19 +115,20 @@ abstract contract StablePoolProtocolFees is StablePoolStorage, StablePoolRates,
         // amplification are not translated into changes to the invariant. Since amplification factor changes are both
         // infrequent and slow, they should have little effect in the pool balances, making this a very good
         // approximation.
-        uint256 lastPostJoinExitAmp = _lastPostJoinExitAmp;
+
+        (uint256 lastJoinExitAmp, uint256 lastPostJoinExitInvariant) = getLastJoinExitData();
 
         uint256 swapFeeGrowthInvariant = StableMath._calculateInvariant(
-            lastPostJoinExitAmp,
+            lastJoinExitAmp,
```

Comment:
I was questioning the name change, but it makes sense. The invariant changes as a result of the join/exit, so it is a "PostJoinExit" thing, whereas the Amp is unrelated - you're just grabbing whatever it happens to be at that time, so it's the Amp "at" the last JoinExit.